wire-ios/Wire-iOS/Sources/Helpers/Date+Format.swift

```diff
@@ -35,15 +35,11 @@ public class WRDateFormatter {
         return timeFormatter
     }()
 
-    static var timeIntervalFormatter: TTTTimeIntervalFormatter = {
-        let locale = Locale.current
-        let timeFormatter = TTTTimeIntervalFormatter()
-        timeFormatter.presentTimeIntervalMargin = 60
-        timeFormatter.locale = locale
-        timeFormatter.usesApproximateQualifier = false
-        timeFormatter.usesIdiomaticDeicticExpressions = true
-
-        return timeFormatter
+    static var timeIntervalFormatter: RelativeDateTimeFormatter = {
+        let formatter = RelativeDateTimeFormatter()
+        formatter.unitsStyle = .full
+        formatter.dateTimeStyle = .named
+        return formatter
```

Comment:
@johnxnguyen Do we need to either update or write new tests?

Reply:
I haven't changed any behavior so existing tests are still valid, but I haven't checked how detailed those tests are.